tools-cli/src/main/java/com/hedera/hashgraph/client/cli/options/CollateCommand.java

```diff
@@ -226,47 +221,69 @@ private List<List<String>> verifyTransactions() {
 		// that transaction.
 		final Map<String, List<String>> verifyWithFiles = new HashMap<>();
 
-		for (final var entry : transactions.entrySet()) {
+		final var iterator = transactions.entrySet().iterator();
+		while (iterator.hasNext()) {
+			final var entry = iterator.next();
 			var helper = entry.getValue();
 			// Get the transactionId and use that as the key for the verification map
 			var transactionId = helper.getBaseName();
 			var fileName = helper.getTransactionFile();
-			// Make sure this transaction isn't already in the map
-			// (Multi-node submission will have duplicate transactionId entries)
-			if (verifyWithFiles.containsKey(transactionId)) {
-				continue;
-			}
-
 			// Collate all signatures, this process will also verify the signatures,
 			// ensuring the required signatures are present.
-			helper.collate(infoFiles);
+			try {
+				helper.collate(infoFiles);
+			} catch (HederaClientRuntimeException e) {
+				// If collating failed, add an item to the verification list
+				// Make sure this transaction isn't already in the map
+				// (Multi-node submission will have duplicate transactionId entries)
+				if (!verifyWithFiles.containsKey(transactionId)) {
+					var verificationItemList = new ArrayList<String>();
+					verificationItemList.add(fileName);
+					verificationItemList.add(transactionId);
+					verifyWithFiles.put(transactionId, verificationItemList);
+					logger.info("Collation and Verification of " + transactionId
```

Comment:
Do you think its an info level message or a warn ?

Reply:
Yes, that is probably better. Updated.